node/packages/client/src/cosmos/parser/index.ts

```diff
@@ -33,48 +42,47 @@ export class TransactionParser {
       status: tx.confirmations > 0 ? Status.Confirmed : Status.Pending, // TODO: handle failed case
       transfers: [],
       txid: tx.txid,
-      value: tx.value,
+      value,
+      data: {
+        parser: 'cosmos',
```

Comment:
I grepped for `parser: ` and we currently use strings for parsers but it would be great to have an enum for these. Happy to add it after this is merged!